### ISSUE TITLE
neovim cannot copy to clipboard in tmux, but this can be fixed in 5 min.

### ISSUE THREAD

**Issue by Jim-Zenn** [termux/termux-packages#1828]
Currently, neovim cannot copy to termux clipboard in a tmux.

I have identified the problem. 

tmux has higher priority to get control of neovim clipboard because this patch is not implemented correctly. The patch should check termux first, then check if there is tmux.

https://github.com/termux/termux-packages/blob/master/packages/neovim/runtime-autoload-provider-clipboard.vim.patch

please fix.

**Comment by Jim-Zenn** [comment]
It should be like this.
```
elseif executable('termux-clipboard-set')
    let s:copy['+'] = 'termux-clipboard-set'
    let s:paste['+'] = 'termux-clipboard-get'
    let s:copy['*'] = s:copy['+']
    let s:paste['*'] = s:paste['+']
    return 'termux-clipboard'
  endif
  elseif exists('$TMUX') && executable('tmux')
    let s:copy['+'] = 'tmux load-buffer -'
    let s:paste['+'] = 'tmux save-buffer -'
    let s:copy['*'] = s:copy['+']
    let s:paste['*'] = s:paste['+']
    return 'tmux'
  endif
```

**Comment by fornwall** [comment]
Thanks! This should now be fixed in the latest version `0.2.3~2017.11.25` of the neovim package - run `pkg up` to update.

**Comment by Jim-Zenn** [comment]
No, thank YOU for making neovim and terminal possible on android!!!
